src/wiktextract/extractor/en/page.py

```diff
@@ -1379,6 +1384,27 @@ def process_gloss_header(
     ) -> None:
         ruby = []
         links: list[str] = []
+
+        # process template parse nodes here
+        new_nodes = []
+        info_template_data = []
+        for node in header_nodes:
+            info_data, info_out = parse_info_template_node(wxr, node, "head")
+            if info_data or info_out:
```

Comment:
`info_out` is always `None` when `loc` is `head`, I think `parse_info_template_node()` only need to return `info_data` and remove `+obj` template nodes from `header_nodes` does the same thing.

Reply:
This is the case for `Template:+obj`, but there *might* be other templates whose output we want to keep even in heads.